### PR TITLE
Issue-308

### PR DESCRIPTION
**Pull request by marcelovilaca**
https://github.com/indigo-iam/iam/issues/308

Changed:

/iam/iam-login-service/src/main/java/it/infn/mw/iam/api/tokens/converter/TokensConverter.java -> to not expose the value of the tokens
/iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/RefreshTokenGetRevokeTests.java -> do not throw error in assertion test
/iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/AccessTokenGetRevokeTests.java -> do not throw error in assertion test

### CODE REVIEW

File: iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/AccessTokenGetRevokeTests.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public void getAccessToken() throws JsonParseException, JsonMappingException,
     System.out.println(remoteAt);
 
     assertThat(remoteAt.getId(), equalTo(at.getId()));
-    assertThat(remoteAt.getValue(), equalTo(at.getValue()));
+    assertThat(remoteAt.getValue(), equalTo(null));
```
Comment (andreaceccanti) [inline iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/AccessTokenGetRevokeTests.java]: There's a better matcher to check for null values, IIRC is called  `nullValue()`

File: iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/RefreshTokenGetRevokeTests.java [hunk 2]
```diff
@@ -100,7 +100,7 @@ public void getRefreshToken() throws JsonParseException, JsonMappingException,
     System.out.println(remoteRt);
 
     assertThat(remoteRt.getId(), equalTo(rt.getId()));
-    assertThat(remoteRt.getValue(), equalTo(rt.getValue()));
+    assertThat(remoteRt.getValue(), equalTo(null));
```
Comment (andreaceccanti) [inline iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/RefreshTokenGetRevokeTests.java]: See above

File: iam-common/pom.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>it.infn.mw</groupId>
     <artifactId>iam-parent</artifactId>
-    <version>1.6.0-issue-308-SNAPSHOT</version>
+    <version>1.6.1-issue-308</version>
```
Comment (andreaceccanti) [inline iam-common/pom.xml]: There's no need to increase the version # to 1.6.1, 1.6.0-issue-308 (with or without SNAPSHOT) is fine

File: iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/AccessTokenGetRevokeTests.java (hunk 1 above)
Comment (marcelovilaca) [reply iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/AccessTokenGetRevokeTests.java]: Fixed, 
I have a question about the usage of this matcher regarding the code efficiency.. for my education purpose only... we talk when you're back :)

**Pull request merged**